lib/opencensus/stats/recorder.rb

```diff
@@ -9,9 +9,7 @@ module Stats
     # Stats recorder.
     #
     # Recorder record measurement against measure for registered views
-    # Span represents a span in a trace record. Spans are contained in
-    # a trace and arranged in a forest. That is, each span may be a root span
-    # or have a parent span, and may have zero or more children.
+    # Span represents a span in a trace record.
```

Comment:
You're still talking about spans. Where are spans used in this class?

Reply:
Removed span.